recipes/redis-plus-plus/all/conanfile.py

```diff
@@ -18,12 +18,14 @@ class RedisPlusPlusConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
-        "with_tls": [True, False]
+        "with_tls": [True, False],
+        "enable_async": [False, "libuv"]
     }
     default_options = {
         "shared": False,
         "fPIC": True,
-        "with_tls": False
+        "with_tls": False,
+        "enable_async": "libuv"
```

Comment:
```suggestion
        "enable_async": "libuv"
```

just `async` maybe?

Reply:
trying to follow https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#recommended-feature-options-names

Reply:
async is a reserved keyword on it's own so it causes syntax errors